src/screens/tabs/Profile/screens/Settings/components/SettingItem/index.js

```diff
@@ -20,7 +20,7 @@ const SettingItem = ({ icon, name, description, onPress, iconName }) => (
         <Text style={[FontStyles.Normal, styles.name]}>{name}</Text>
         <Text style={FontStyles.NormalGray}>{description}</Text>
       </Column>
-      <Icon name="chevronRight" style={styles.chevron} />
+      <Icon name="chevronRight" style={styles.chevron} height={18} />
```

Comment:
why not height inside the  style?

Reply:
it doesnt work in SVGs, they need to be sent as props